<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="RootLayout">
   <widget class="QWidget" name="Main" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QPushButton" name="StartBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>430</y>
       <width>160</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>210 썬플라워 B</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="TacLab">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>130</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>210 썬플라워 B</family>
       <pointsize>50</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Tac</string>
     </property>
    </widget>
    <widget class="QLabel" name="ToeLab">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>210 썬플라워 B</family>
       <pointsize>50</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Toe</string>
     </property>
    </widget>
    <widget class="QLabel" name="TicLab">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>210 썬플라워 B</family>
       <pointsize>50</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Tic</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Mode" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>381</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>210 썬플라워 B</family>
       <pointsize>31</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select Game Mode</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPushButton" name="play1">
      <property name="font">
       <font>
        <family>210 썬플라워 B</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Computer VS Player</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="comVsplay" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QLabel" name="NameLabelP1">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>60</y>
       <width>400</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>210 썬플라워 B</family>
       <pointsize>27</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter a Player Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PlayLabel1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>210 썬플라워 B</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Player</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>500</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>210 썬플라워 B</family>
       <pointsize>34</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PLAY!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Game" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>133</width>
       <height>133</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>90</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>133</width>
       <height>133</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>90</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>133</width>
       <height>133</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>90</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>133</width>
       <height>133</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>90</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>300</y>
       <width>133</width>
       <height>133</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>90</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn6">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>300</y>
       <width>133</width>
       <height>133</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>90</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>133</width>
       <height>133</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>90</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>430</y>
       <width>133</width>
       <height>133</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>90</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn9">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>430</y>
       <width>133</width>
       <height>133</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>90</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="P1">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>62</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Player</string>
     </property>
    </widget>
    <widget class="QLabel" name="P2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Computer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>O</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Result" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QLabel" name="winText">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>401</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>210 썬플라워 B</family>
       <pointsize>25</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text">
      <string>Player1 WIN!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>400</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>210 썬플라워 B</family>
       <pointsize>31</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
